src/Polly.Net35/CircuitBreaker/SuccessRatioCircuitBreakerState.cs

```diff
@@ -47,6 +53,7 @@ public void Reset()
             using (TimedLock.Lock(_lock))
             {
                 _successCount += 1;

```

Comment:
Then I assume the currentSuccessRatio calculation should not use decayed values, since decay is done before incrementing?